pyteal/ast/txn_test.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+from pyteal.ast.gitxn import Gitxn
```

Comment:
This can be removed by adding `Gitxn` to the imports in `pyteal/ast/__init__.py`